databases/week2/Homework/Homework_tables.sql

```diff
@@ -0,0 +1,152 @@
+USE Clinic_scheduling;
+
+CREATE TABLE
+  Clinics (
+    clinic_id INT NOT NULL AUTO_INCREMENT,
+    Clinic_name VARCHAR NOT NULL,
+    Clinic_address VARCHAR NOT NULL,
+    PRIMARY KEY (clinic_id)
+  );
+
+-- a USER can either have work schedule for certain date or a regular working schedule for certain weekdays
+-- hence different kind of time-slots
+CREATE TABLE
+  Working_hours (
+    date DATE,
+    weekday CHAR(3),
+    begins DATETIME NOT NULL,
+    ends DATETIME NOT NULL,
+    slot_id INT NOT NULL AUTO_INCREMENT,
+    PRIMARY KEY (slot_id)
+  );
+
+CREATE TABLE
+  Users (
+    user_id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR NOT NULL,
+    sex VARCHAR,
+    address VARCHAR,
+    phone INT NOT NULL,
+    email VARCHAR NOT NULL,
+    notes VARCHAR,
+    PRIMARY KEY (user_id)
+  );
+
+CREATE TABLE
+  Medical_forms (
+    Form_id INT NOT NULL AUTO_INCREMENT,
+    Form_name VARCHAR NOT NULL,
+    Form_link VARCHAR NOT NULL,
+    Form_description VARCHAR NOT NULL,
+    PRIMARY KEY (Form_id)
+  );
+
+-- each doctor and several other user roles have specific schedules
+CREATE TABLE
+  Doctors_have_working_hours (
+    schedule_id INT NOT NULL AUTO_INCREMENT,
+    slot_id INT NOT NULL,
+    user_id INT NOT NULL,
+    PRIMARY KEY (schedule_id),
+    FOREIGN KEY (slot_id) REFERENCES Working_hours (slot_id),
```

Comment:
it is a good practice for the primary key to be called `id` (unless it has specific meaning, like CPR, or barcode, etc) and a firing key to be `<table>_id`
this comment is for all tables you created
```suggestion
    FOREIGN KEY (working_hour_id) REFERENCES Working_hours (slot_id),
```

Reply:
Changed all PKs to "id"